envydis/gk110.c

```diff
@@ -1654,7 +1654,7 @@ static struct insn tabc[] = {
 	{ 0x1b80000000000000ull, 0xff80000000000000ull, N("quadon") },
 	{ 0x1c00000000000000ull, 0xff80000000000000ull, N("quadpop") },
 
-	{ 0x2000000000000000ull, 0xfc80000000000000ull, T(join), T(p), T(logop38), N("b32"), DST, SRC1, LIMM },
+	{ 0x2000000000000000ull, 0xf880000000000000ull, T(join), T(p), T(logop38), N("b32"), DST, T(not3a), SRC1, LIMM },
```

Comment:
While correct, you could go a step further...

$ perl -ane 'foreach (@F) { print pack "I", hex($_) }' > tt; nvdisasm -b SM35 tt
0 28000000
        .headerflags    @"EF_CUDA_SM35 EF_CUDA_PTX_SM(EF_CUDA_SM35)"
        /*0000*/               @P0 LOP32I.AND R0, R0, 0x0.INV;

$ perl -ane 'foreach (@F) { print pack "I", hex($_) }' > tt; nvdisasm -b SM35 tt
0 20800000
        .headerflags    @"EF_CUDA_SM35 EF_CUDA_PTX_SM(EF_CUDA_SM35)"
        /*0000*/               @P0 LOP32I.AND R0.CC, R0, 0x0;

$ perl -ane 'foreach (@F) { print pack "I", hex($_) }' > tt; nvdisasm -b SM35 tt
0 30000000
        .headerflags    @"EF_CUDA_SM35 EF_CUDA_PTX_SM(EF_CUDA_SM35)"
        /*0000*/               @P0 LOP32I.AND.X R0, R0, 0x0;

So actually the mask should probably be 0x2000000000000 and add the not3b before the LIMM and an acout37 for the .CC flag (placed after DST), and acin3c at the end for the .X.